src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/Client/App.razor

```diff
@@ -0,0 +1,12 @@
+﻿<Router AppAssembly="@typeof(App).Assembly">
+    <Found Context="routeData">
+        <RouteView RouteData="@routeData" DefaultLayout="@typeof(MainLayout)" />
+        <FocusOnNavigate RouteData="@routeData" Selector="h1" />
+    </Found>
+    <NotFound>
+        <PageTitle>Not found</PageTitle>
+        <LayoutView Layout="@typeof(MainLayout)">
+            <p role="alert">Sorry, there's nothing at this address.</p>
+        </LayoutView>
+    </NotFound>
+</Router>
```

Comment:
Is the intent to support auth with this template? I don't personally mind either way about that, but if we are supporting auth, then I think there will also need to be an auth-enabled version of this file which uses `AuthorizeRouteView`. The non-empty template contains an example of this.

Reply:
> Is the intent to support auth with this template?

[Spoke with Dan](https://github.com/dotnet/aspnetcore/issues/32832#issuecomment-1137809381) regarding this earlier, and the conclusion was the empty template does not need to support auth.